bin/brew

```diff
@@ -97,6 +97,14 @@ then
   export CI="1"
 fi
 
+# Backup specific variables for shellenv.
+for VAR in PATH MANPATH INFOPATH FPATH; do
```

Comment:
`HOMEBREW_PATH` is already set. I'd suggest using the same logic for `MANPATH`, `INFOPATH`, `FPATH`

Reply:
> I'd suggest using the same logic for `MANPATH`, `INFOPATH`, `FPATH`.

Should I put these variables `global.rb`?

https://github.com/Homebrew/brew/blob/bdc3391a1319f1c40a108d14a077ca0964b33a3f/Library/Homebrew/global.rb#L116-L123